gui/default/index.html

```diff
@@ -329,9 +332,39 @@ <h3 class="panel-title">
         </div>
       </div>
 
-      <!-- First regular row -->
+      <!-- Login form -->
+      <div ng-if="!authenticated" class="center-block">
+        <h3 translate>Authentication required</h3>
 
-      <div class="row">
+        <form ng-submit="authenticatePassword()">
+          <div class="row form-group">
+            <label for="Username" translate>Username</label>
+            <input id="Username" class="form-control" type="text" ng-model="login.username" />
+          </div>
+
+          <div class="row form-group">
+            <label for="Password" translate>Password</label>
+            <input id="Password" class="form-control" type="password" ng-model="login.password" ng-trim="false" />
```

Comment:
Isn't it customary to use all-lowercase for `id` stuff?

Reply:
This is still inconsistent in the latest iteration, you're now using `user` as ID in lowercase, but `Password` capitalized.

Reply:
Actually I meant the other way, all lowercased. If you search through the file for id attributes, they're all lowercase.

Reply:
I've chosen for now to use `id="User"` and `id="Password"`, which matches the existing IDs in [`SettingsModalView.html`](https://github.com/syncthing/syncthing/blob/462389934b87eb1b2e2deb8b2b19cc504b449c3a/gui/default/syncthing/settings/settingsModalView.html#L132-L138). This does indeed not match the style in the rest of this file, but it seems more appropriate to me to align with the style used in a related section of code, even if in a different file.

Reply:
I'd still call it a mistake in the settings code to use non-lowercase IDs, which shouldn't be proliferated.  The real question though is how do e.g. password managers handle the fields? Surely the settings view benefits from using the exact same spelling as here. Feel free to adjust in both places and make it consistent, lowercase.

Reply:
Ok, done!